src/schema.ts

```diff
@@ -147,23 +147,56 @@ type Topic {
   supplementaryResources: [Resource]
 }
 
+type FrontpageSubjects {
+  category: String
+  subjects: [String]
+}
+
+type Frontpage {
+  topical: [String]
+  subjects: [FrontpageSubjects]
+}
+
+type SubjectPageArticles {
+  location: String
+  articleIds: [String]
```

Comment:
Should be renamed to to resources and return `[Resource]`. Resource type should be expanded with `meta: ArticleSubset`